hub/components/EditRealmConfig/Summary/index.tsx

```diff
@@ -45,11 +45,11 @@ export function Summary(props: Props) {
       />
       <ProposalVoteType
         className="mt-8"
-        currentCommunityRules={props.governance.communityTokenRules}
-        currentCouncilRules={props.governance.councilTokenRules}
-        currentBaseVoteDays={props.governance.maxVoteDays}
-        currentCoolOffHours={props.governance.coolOffHours}
-        currentMinInstructionHoldupDays={
+        initialCommunityRules={props.governance.communityTokenRules}
```

Comment:
i refactored all "current" prop names to "initial" because this is more generic while also being, imo,as precise or more.